### ISSUE TITLE
Failed to convert token to runtime object: True

### ISSUE THREAD

**Issue by bkmeneguello** [ephread/inkgd#31]
Testing this ink file:
```
VAR test = true
Hello {test:world!}
```
compiled with inkcatle v1.0.0 gives me `Failed to convert token to runtime object: True`.
[This is the line](https://github.com/ephread/inkgd/blob/3127d994106fa00e93430a9153f1b2fa3cc287ae/addons/inkgd/runtime/static/json.gd#L422) where the error happens. I changed [this line](https://github.com/ephread/inkgd/blob/3127d994106fa00e93430a9153f1b2fa3cc287ae/addons/inkgd/runtime/static/json.gd#L256) to
```
if token is int || token is float || token is bool:
```
And it worked.

**Comment by ephread** [comment]
Hey @bkmeneguello same as #30, inkgd doesn't support stories compiled with inklecate 1.0.0 yet. It's coming in the future!

**Comment by ephread** [comment]
Fixed by #49.

**Comment by ephread** [comment]
Fixed by #49.

**Comment by ephread** [comment]
Fixed by #49.
